common/components/WalletDecrypt/components/WalletButton.tsx

```diff
@@ -86,24 +86,47 @@ export class WalletButton extends React.PureComponent<Props> {
       >
         <div className="WalletButton-inner">
           <div className="WalletButton-title">
-            {icon && <img className="WalletButton-title-icon" src={icon} />}
+            {icon && <img className="WalletButton-title-icon" src={icon} alt="Logo" />}
```

Comment:
This alt should probably be dynamic, e.g. `alt={'${name} Logo'}`

Reply:
I tried doing `alt={'${name} Logo'}` but it felt a bit weird because I'd get something like 'bitcoin logo bitcoin' which just felt off. Personally I think just `alt="logo"` is the best option